src/main/java/org/myrobotlab/service/meta/SolrMeta.java

```diff
@@ -16,8 +16,8 @@ public SolrMeta() {
 
     addDescription("Solr Service - Open source search engine");
     addCategory("search");
-    String solrVersion = "8.11.2";
-    String luceneVersion = solrVersion;
+    String solrVersion = "9.2.0";
+    String luceneVersion = "9.4.2";
```

Comment:
When I updated Solr on my own branch I needed to add the following here:

```
addDependency("org.apache.solr", "solr-scripting", solrVersion);
exclude("com.google.guava", "*");
```

From what I remember, one of the classes used in the solrconfig.xml was moved to this artifact, but I can't remember off the top of my head which one. Do you think this is necessary? I can find which class it was by looking through a diff of my solconfig.xml if you need more info

Reply:
Nevermind, looks like it was `XSLTResponseWriter` which your changes have removed from the solrconfig.xml